src/com/Jacksonnn/DCCore/DCCore.java

```diff
@@ -119,13 +140,12 @@ private void registerListeners() {
 		pm.registerEvents(new SpawnerListener(), this);
 		pm.registerEvents(new QuickDepositListener(), this);
 		pm.registerEvents(new ChatListener(), this);
-        pm.registerEvents(new ResponseListener(), this);
-        pm.registerEvents(new GuestQuizListener(), this);
-        pm.registerEvents(new onChatEvent(), this);
-        pm.registerEvents(new StaffNotification(), this);
-        pm.registerEvents(new Vanish(), this);
-        pm.registerEvents(new BannedWordsListener(), this);
-		pm.registerEvents(new DCPlayerListener(), this);
```

Comment:
Why was the register for DCPlayerListener moved away from all the other listeners? Also, thanks for fixing the whitespace, don't know how it slipped from my fingers, but GuestQuizListener was pushed up a line